test/e2e/traces_test.go

```diff
@@ -0,0 +1,231 @@
+//go:build integration
+// +build integration
+
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/efficientgo/e2e"
+	"github.com/efficientgo/tools/core/pkg/backoff"
+	"github.com/efficientgo/tools/core/pkg/testutil"
+)
+
+const (
+	// Note that if the forwarding collector uses OIDC flow instead of hard-coding
+	// the bearer token we would need
+	// "otel/opentelemetry-collector-contrib:0.45.0" instead.
+	otelFwdCollectorImage = "otel/opentelemetry-collector:0.45.0"
+
+	// OTel trace collector that receives in HTTP w/o security, but exports in gRPC with security.
+	otelForwardingConfig = `
```

Comment:
What do you think about moving these to `config.go` and moving the forwarder initialization method to `services.go`? It's a bit more natural fit at least for me.